e2e/detail.test.ts

```diff
@@ -0,0 +1,24 @@
+import { by, device, element, expect } from 'detox';
+
+describe('Detail', () => {
+    beforeAll(async () => {
+        await device.launchApp();
+    });
+
+    beforeEach(async () => {
+        await device.reloadReactNative();
+    });
+
+    it('should display detail screen', async () => {
+        await element(by.id('Menu')).tap();
+        await element(by.id('Detail')).tap();
```

Comment:
Change this to `AlphabetDetailLink` or `AlphabetDetailLinkButton`

Reply:
Also, you might want to navigate to the detail screen in the before each since you're doing this in every test.